File: memo-rest/src/main/java/com/jetsbee/memo/domain/Person.java
package com.jetsbee.memo.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Data;

@Data
@Entity
public class Person {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private long id;

	private String firstName;
	private String lastName;
	private String description;
	
	private Person() {}
	
	public Person(String firstName, String lastName, String description) {
		this.firstName = firstName;
		this.lastName = lastName;
		this.description = description;
	}
}

File: memo-rest/src/main/resources/static/components/Person.js
import React, { PropTypes, Component } from 'react'

class Person extends Component {
	render() {
		return (
			<tr>
				<td>{this.props.person.firstName}</td>
				<td>{this.props.person.lastName}</td>
				<td>{this.props.person.description}</td>
			</tr>
		)
	}
}

Person.propTypes = {
  person: PropTypes.object.isRequired
}

export default Person
